dashboard/app/views/levels/editors/_pythonlab.html.haml

```diff
@@ -1,5 +1,6 @@
 = render partial: 'levels/editors/fields/long_instructions', locals: {f: f}
 = render partial: 'levels/editors/fields/teacher_only_markdown', locals: {f: f}
+= render partial: 'levels/editors/fields/video', locals: {f: f}
```

Comment:
Similarly, I wondered if this video field should be under the help & tips heading? But I wanted to be cautious about mucking around too much in a way that could impact other level editing. 🤔

Reply:
Yeah I think it's fine to follow the existing pattern here, even though it would make sense to put it under help and tips